GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadListViewModel.swift

```diff
@@ -108,10 +109,15 @@ extension SecureConversations {
 
 extension SecureConversations.FileUploadListViewModel {
     var maxUnscrollableViews: Int {
-        if environment.uiApplication.preferredContentSizeCategory() <= .accessibilityMedium {
-            return .maxUnscrollableViewsOnDefaultContentSizeCategory
-        } else {
-            return .maxUnscrollableViewsOnLargeContentSizeCategory
+        switch environment.scrollingBehaviour {
+        case .nonScrolling:
+            return .max
+        case let .scrolling(application):
+            if application.preferredContentSizeCategory() <= .accessibilityMedium {
```

Comment:
Would be nice to know the logic behind this. At least for me it's not obvious.

Reply:
Added documentation and comments 04c21f51df42d6893d9a52162669ad35a94f559b.